sqlserver/datadog_checks/sqlserver/statements.py

```diff
@@ -379,6 +379,14 @@ def _load_plan(self, plan_handle, cursor):
             return None
         return result[0][0]
 
+    @staticmethod
+    def _get_avg_query_duration(row):
```

Comment:
Need a test specifically for this to confirm that the latency conversion works for python2. I think we need to convert one of those numbers to a float else it'll do integer division in python2.

Reply:
c4a28264881bc882cea023e6a73fa2edcfbadccd